commands/bmc.nicole

```diff
@@ -20,17 +20,12 @@ NETWORK_CLIENT_IFACE="xyz.openbmc_project.Network.Client"
 # Format date in a non-standard way
 function format_date {
   local compact="$1"
-  local utc="$2"
   local sec="$3"
 
-  if [[ -n "${compact}" ]] && [[ -n "${utc}" ]]; then
```

Comment:
I can't understand this change, Is it will be 
```bash
local sec="$3"

elif [[ -n "${compact}" ]]; then
    date --date "@${sec}" +'%F %T UTC%z'
...
```
It seems to me that this is incorrect. This should be at least

```bash
if [[ -n "${compact}" ]]; then
    date --date "@${sec}" +'%F %T UTC%z'
    
```

Reply:
Thanks. Typo indeed. Changed in one place and forgot here. It must be `if`.

Reply:
Fixed it